.github/workflows/caches.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
   },
   npm: {
     path: [`${homeDirectory}/.npm`],
-    hashFiles: ['superset-frontend/package-lock.json'],
+    hashFiles: [`${workspaceDirectory}/superset-frontend/package-lock.json`],
```

Comment:
This cache was disabled but we are keeping the configuration anyway, in case there are other ways to make cache loading faster in the future.

The change here is to make sure we hash the same files wherever the cache step runs.